tutorials/eve7/collection_proxies.C

```diff
@@ -790,6 +809,11 @@ void collection_proxies(bool proj=true)
    auto event = new Event();
    event->Create();
 
+   // divert selection to map proxy builder products with collection
+   auto deviator = std::shared_ptr<FWSelectionDeviator>(new FWSelectionDeviator());
```

Comment:
Better

    auto deviator = std::make_shared<FWSelectionDeviator>();

Reply:
@linev I used  std::make_shared for instantiation and removed unnecessary nullptr.